.github/workflows/sycl-containers.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
     if: github.repository == 'intel/llvm'
     name: Base Ubuntu 22.04 Docker image
     runs-on: ubuntu-22.04
+    permissions:
+      packages: write
```

Comment:
You provided write permission to all jobs in the workflow, so it's better to change the workflow permission at line 24 instead.

Reply:
This is intended, to follow the `OSSF` recommendation.

See https://github.com/intel/llvm/pull/13173#issue-2209848734

```
per OSSF (https://securityscorecards.dev/viewer/?uri=github.com/intel/llvm) all workflows should have default top level permission set. Which we set to below as per recommendation

permissions:
contents: read

then within actual jobs, when needed, we added additional privileges.
```